frontend/components/product-list/sections/FilterableProductsSection/ProductListFilters/FilterRow.tsx

```diff
@@ -309,3 +319,25 @@ function avg(x: string): number | null {
    }
    return nums.reduce((x, y) => x + parseFloat(y), 0) / nums.length;
 }
+
+function compareCapacity(a: FacetOption, b: FacetOption): number {
+   return capacityStringToBytes(b.value) - capacityStringToBytes(a.value);
+}
+
+const unitToBytes = {
+   'B': 1,
+   'KB': 1024,
+   'MB': 1024 * 1024,
+   'GB': 1024 * 1024 * 1024,
+   'TB': 1024 * 1024 * 1024 * 1024,
+   'PB': 1024 * 1024 * 1024 * 1024 * 1024,
+};
```

Comment:
```suggestion
const unitToBytes = {
   'B': 1,
   'KB': 1024,
   'MB': 1024 ** 2,
   'GB': 1024 ** 3,
   'TB': 1024 ** 4,
   'PB': 1024 ** 5,
};
```

Is there a reason the exponent operator isn't being used here? Seems a bit easier to read/write.

Reply:
Because Github co-pilot suggested this and I was so enamored that it got it correct that I wanted to just leave it :-)

That said, your version is better I think.

Reply:
I was curious about this because I had a half memory that there was an exp operator but we didn't use it for some reason. Turns out it had limited support until relatively recently. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Exponentiation

Probably not a big deal now, especially if it gets transpiled, but it doesn't work on any version of IE or node 6, so we probably used to have a rule not to use it when everything server-side was on node 6 and everything client-side had to work on IE.